### DISCUSSION TITLE
Code completion for table names & column names.

### DISCUSSION THREAD

**Discussion by lucabarletta** [caioricciuti/ch-ui#12] [Ideas]
Is this feature planed for future?

**Comment by caioricciuti** [comment]
Hey @lucabarletta, yep so I'm working in the V2 that have multiple new features, one of them being the enhanced intelli-sense, both for DB tables as built in functions and so on. I can't give you a release date or anything because I'm working by my own and on my free time. Also the new version have user management, credentials and an easier way to work with your data! Stay tuned and thank you for using CH-UI!

**Reply by lucabarletta** [reply]
Thank you for the implementation!

**Comment by caioricciuti** [comment]
Hey I'll close this discussion since we have it implemented... Thank you!
